fe/fe-core/src/main/java/org/apache/doris/planner/external/jdbc/JdbcFunctionPushDownRule.java

```diff
@@ -168,9 +171,48 @@ private static String replaceFunctionNameIfNecessary(String func, Predicate<Stri
             }
             if (newFunc != null) {
                 functionCallExpr.setFnName(FunctionName.createBuiltinName(newFunc));
-                func = functionCallExpr.getFnName().getFunction();
             }
         }
-        return func;
+    }
+
+    // Function used to convert nereids planner's function to old planner's function
+    private static Expr replaceGenericFunctionExpr(FunctionCallExpr functionCallExpr, String func) {
+        Map<String, String> supportedTimeUnits = Maps.newHashMap();
```

Comment:
This should a static field?

Reply:
replaceGenericFunctionExpr is static because it processes based on parameters without requiring object state, and it's used in static contexts where non-static methods cannot be called directly.